src/ashwini/assignment24/ReturnMiddleCharOfString.java

```diff
@@ -0,0 +1,44 @@
+package ashwini.assignment24;
+/*
+ 
+Assignment - 24 : 18th April'2021
+
+Create a method which will return middle character of each String. Print output in main method.
+String[] arr = {"Techno", "Hello", "Credits"};
+output : c l d
+
+Note:  for name having even length, consider length/2-1 index is middle character.
+
+ */
+
+public class ReturnMiddleCharOfString {
+	char printMidCharOString(String str) {
+		char middleChar = ' ' ;
+		for(int index = 0; index<str.length(); index++) {
```

Comment:
Take reference of this
Sandesh - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1063/files